index.html

```diff
@@ -0,0 +1,211 @@
+          Pharmcists in the Pharmacy space, the ACPN conference provides a
+          focused and unique opportunity to learn, share best practices, network
+          with peers, and interact with other healthcare providers in the
+          pharmaceutical space. At the ACPN Conference, community Pharmacists
+          can learn how best to offer pharmaceutical care and gain insights on
+          more efficient drug therapy management.
+        </p>
+        <time datetime="2023-05-20T19:30">June 15, 2023 @ 9:00am</time>
+        <p class="venue">
+          Festival Hotel, along Amuwo Odofin, Festac town, Lagos State, Nigeria.
+        </p>
+      </section>
+    </main>
+    <!------------------------------MAIN  END------------------------------------->
+
+    <!------------------------------PROGRAMS STARTS------------------------------->
+    <section class="events">
+      <div class="holder">
+        <h2>Main Program</h2>
+        <hr />
+        <div class="events-content">
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-analysis"></i>
+              <h3>Exhibition</h3>
+            </div>
+            <p>
+              40+ Pharmaceutical & FMCG companies have confirmed their
+              participation as exhibitors at the conference.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-tv-retro"></i>
+              <h3>Excursion</h3>
+            </div>
+            <p>
+              Explore the vast Historic collections at the national Museum, Walk
+              through a range of art works at the Nike Art Garlary
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-comment-alt-notes"></i>
+              <h3>Workshop</h3>
+            </div>
+            <p>
+              The mandatory continuing professional development program that
+              will be hosted in this year’s conference is focused on Vaccination
+              Training for Pharmacists.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-presentation-edit"></i>
+              <h3>Novelty Match</h3>
+            </div>
+            <p>
+              There will be a Novelty Match between the different state arm of
+              the association.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon">
+              <i class="uil uil-palette"></i>
+              <h3>Group Dynamics</h3>
+            </div>
+            <p>
+              Every state arm of the association will be required to exhit or
+              showcase their progress and activites within the past one year
+            </p>
+          </div>
+          <div class="join">
+            <a href="#">Join ACPN 2023.</a>
```

Comment:
- Good job done so far, but there are some changes required, in the mobile version for this section, `individual articles have their elements organized in an horizontal row, with the image at the left, title in the center, and description at the right`. 
Your project:
![dfzfnzdn](https://user-images.githubusercontent.com/73607512/234030461-a7f3decb-c6ae-40c0-921b-b35d121ede2b.PNG)

Design template:
![rhaerh](https://user-images.githubusercontent.com/73607512/234030724-205e04d5-39f1-46f5-9200-62ab1645f160.PNG)